source/elements/oneDPL/source/parallel_api.rst

```diff
@@ -262,6 +262,19 @@ Iterators
 The oneDPL iterators are defined in the :code:`<oneapi/dpl/iterator>` header,
 in :code:`namespace oneapi::dpl`.
 
+Let us define a named requirement, :code:`ValidParallelIteratorSource`, to describe valid random access iterator-like
```

Comment:
I think the word "valid" is redundant in the name; it really adds nothing to the meaning. And I do not know how to interpret "ParallelIterator".

I would suggest something like `AlteringIteratorSource` or maybe `Adjusting` or `Adapting` (or whatever English adjective describes best what transform. permutation and zip iterators do with their sources).

Reply:
I can think about this, and propose a better name.